android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -263,7 +264,7 @@ class AddonsManagerAdapter(
             addonsManagerDelegate.onAddonItemClicked(addon)
         }
 
-        holder.addButton.isVisible = !addon.isInstalled()
+        holder.addButton.isInvisible = addon.isInstalled()
```

Comment:
Noticed when the add-on is installed, we don't have any space between the left gutter, and when the text long it looks cramped.

Without this fix.
![screenshot-1696634560492](https://github.com/mozilla-mobile/firefox-android/assets/773158/ff7e6ddf-af5c-4d8f-bd48-758724bbd8a2)

With this fix, this ensures that we always have some space between the add ui and the left gutter.
![image](https://github.com/mozilla-mobile/firefox-android/assets/773158/1f7aa8fe-f0be-4658-8155-5266ede6a35e)

Reply:
Excellent, thanks!